eslint-config-5minds-typescript/index.js

```diff
@@ -0,0 +1,122 @@
+module.exports  = {
+  'extends': require.resolve('eslint-config-5minds'),
+  'parser': '@typescript-eslint/parser',
+  'parserOptions': {
+    'ecmaVersion': 2018,
+    'sourceType': 'module'
+  },
+  'plugins': ['@typescript-eslint', '6river'],
+  'rules': {
+    'import/prefer-default-export': 'off',
+    'new-cap': 'off',
+    '6river/new-cap': ['error', {
+      'newIsCap': true,
+      // this allows decorators to start with an upper case letter
+      'capIsNewExceptionPattern': '(@\\w+)'
+    }],
+
+    '@typescript-eslint/adjacent-overload-signatures': ['error'],
+    '@typescript-eslint/array-type': ['error', 'generic'],
+    '@typescript-eslint/ban-types': ['error', {
+      "types": {
+        'Object': "Use {} instead.",
```

Comment:
Wie auch schon in dem anderen PR von mir angemerkt, plädiere ich für `object` statt `Object`.